### ISSUE TITLE
T2I training in the VAE+SigLIP `TunaModel` appears to leak the clean target image into SigLIP conditioning

### ISSUE THREAD

**Issue by Cherishnoobs** [facebookresearch/tuna-2#11]
## Summary
In the VAE+SigLIP variant, for example `configs/model/tuna_2b.yaml` with `_target_: tuna.models.tuna.TunaModel`, the pure text-to-image training path appears to feed a clean target-image latent into the SigLIP conditioning branch while also using the same image to construct the diffusion target.

For a pure T2I sample, the target image should only be used to produce the VAE latent target and the noisy latent `x_t`. The model should condition on the text prompt, timestep, and noisy latent `x_t`; it should not receive an additional clean embedding of the target image through SigLIP, because such a clean target image is not available at inference time.

## Affected Code Path
The issue seems to follow this path:

1. `tuna/data/datasets/ti_dataset.py` emits both `images` and `images_clip` from the same image for T2I samples.

   The relevant behavior is:

   ```python
   out["images"] = image_tensor
   out["images_clip"] = ...
   ```

   For T2I, both are derived from the target image.

2. `tuna/models/tuna.py::TunaModel.forward` uses `images` to construct noisy diffusion latents and velocity targets:

   ```python
   image_latents, t, image_labels, image_masks, image_latents_clean = (
       self.prepare_latents_and_labels(pixel_values, data_type, image_masks)
   )
   ```

   In `prepare_latents_and_labels`, the clean image is VAE-encoded and then passed through the transport sampler to produce `x_t` and `u_t`.

3. In the same forward pass, `images_clip` is also VAE-encoded deterministically:

   ```python
   if images_clip is not None and hasattr(self.tuna_model, "vae"):
       with torch.no_grad():
           clip_input = images_clip.to(weight_type)
           if clip_input.dim() == 4:
               clip_input = clip_input.unsqueeze(2)
           images_clip = self.tuna_model.vae.sample(
               clip_input, deterministic=True
           )
   ```

4. The wrapper then passes both values into the inner model:

   ```python
   logits, loss_ntp, loss_flow, loss_disp = self.tuna_model(
       text_tokens=text_tokens,
       image_latents=image_latents,
       pixel_values=images_clip,
       ...
   )
   ```

5. Inside `_prepare_embeds`, `pixel_values` takes priority over `image_latents` for SigLIP:

   ```python
   siglip_input = pixel_values if pixel_values is not None else image_latents
   image_embeds_siglip = self.encode_pixels_with_siglip(siglip_input.to(dtype), ...)
   image_embeds = self.siglip_proj(image_embeds_siglip)
   ```

Therefore, for pure T2I training, SigLIP appears to receive a clean deterministic target latent instead of the noisy latent `x_t`.

## Impact
This likely affects T2I training quality for the VAE+SigLIP `TunaModel` variant. In particular, it can make training loss misleading because the model sees target information that is unavailable during generation.

The symptom can be:

- Flow loss decreases normally.
- Teacher-forcing or one-step velocity probes may look reasonable.
- Free T2I sampling still produces noisy or poor images.

This issue may also affect any checkpoint trained with this path, because the learned behavior depends on a training-only shortcut.
